h/templates/accounts/login.html.jinja2

```diff
@@ -1,5 +1,40 @@
-{% extends "h:templates/accounts/base.html.jinja2" %}
+{% extends "h:templates/layouts/base.html.jinja2" %}
 
 {% block page_title %}Log in{% endblock %}
 
-{% set signup_message = "Don't have a Hypothesis account?" %}
+{% block header %}
+  {% include "h:templates/includes/logo-header.html.jinja2" %}
+{% endblock %}
+
+{% block styles %}
+  {{ super() }}
+
+  {# Avoid a "flash of unstyled content" by preloading the stylesheets that
+     will be used by the Preact app within its Shadow DOM. #}
+  {% for url in asset_urls('forms_css') %}
+    <link rel="preload" href={{ url }} as="style"/>
+  {% endfor %}
+{% endblock %}
+
+{% block content %}
+  <div class="form-container">
+    <script type="application/json" class="js-config">{{ js_config|tojson }}</script>
+
+    <h1 class="form-header">Log in</h1>
+    <div id="login-form"></div>
+    <footer class="form-footer">
+      <span style="font-size: 14px">
```

Comment:
Can you set the style to `0.875rem` and add a comment that this aligns with Tailwind's `text-sm`. See https://tailwindcss.com/docs/font-size#basic-example.